src/app/[locale]/(auth)/dashboard/page.tsx

```
@@ -16,12 +48,131 @@ const DashboardIndexPage = () => {
+                    $
+                    {donation.amount}
+                  </span>
+                  <span>{donation.date}</span>
+                </div>
+              </li>
+            ))}
+          </ul>
+        </CardContent>
+      </Card>
+
+      {editingProject && (
```

ESLint

[error] Expected indentation of 8 spaces but found 6.

(style/indent)